test/cli/project-init/ok-interactive/interaction.txt

```diff
@@ -0,0 +1,26 @@
+< Please enter Keboola Storage API host, eg. "connection.keboola.com".
+> %%TEST_KBC_STORAGE_API_HOST%%
+
+< Please enter Keboola Storage API token. The value will be hidden.
+> %%TEST_KBC_STORAGE_API_TOKEN%%
+
+< Please select which project's branches you want to use with this CLI.
+# Select all branches
+> <down arrow>
+> <enter>
+
+< Generate workflows files for GitHub Actions? (Y/n)
+> Y
+
+< Generate "validate" workflow?
+> Y
+
+< Generate "push" workflow?
+> Y
+
+< Generate "pull" workflow?
+> Y
+
+< Please select the main GitHub branch name
+# Select "main"
+> <enter>
```

Comment:
A tu je uz implementovany test.

Ak chcete vidiet podrobnosti, co to robi:
`TEST_VERBOSE=true go test -race -v -count 1 ./test/cli -run TestCliE2E/project-init/ok-interactive`

![image](https://user-images.githubusercontent.com/19371734/191790754-adf9d9d6-1d37-40ae-b3f0-a9eca7f0e0b6.png)